templates/SilverStripe/View/Shortcodes/EmbedShortcodeProvider_video.ss

```diff
@@ -3,6 +3,6 @@
 >
     {$Content}
     <% if $Arguments.caption %>
-        <p class="caption">{$Arguments.caption}</p>
+        <p class="caption">{$Arguments.caption.RAW}</p>
```

Comment:
Why is this changing?

Wouldn't .RAW allow XSS to be added to the caption field and then executed by whoever looks at it?

Reply:
> What happens if the caption contains an `&amp;`?

Sorry, I haven't replied. I added this to avoid double encoding for `&` in caption in Embed short codes. 
We convert all characters to HTML entities before save data in DB. 
I'm not sure 100%, but I check another fields where we use `RAW` method and I also did few XSS tests with string contains `javascript`. 

Just in case, return this back to encode string again: https://github.com/silverstripe/silverstripe-framework/pull/11243/files#diff-d0d2867af6bff9ad7e4ced04d5491feea4b4efec408b6e96b37e4f5eb46914ecR174-R175.
It's not necessary, probably.